string[] class_names

uint32 height # Row count
uint32 width # Column count

uint16Â­[] class_indexes # Row-major order
